CHANGELOG.md

```diff
@@ -5,7 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-Nothing currently
+### Added
+- UsernamePassword credentials are now supported through the use of a `username` tag on the secret
```

Comment:
Apologies this changelog file has not been used since pre 1.0, all releases are tracked on github releases from 1.0 onwards, I've deleted this file on the master branch

Reply:
No worries, updated.